test/e2e/commands/one-shot-single.test.ts

```diff
@@ -64,7 +67,7 @@ const endToEndTestSuite: EndToEndTestSuite = new EndToEndTestSuiteBuilder()
       // TODO pass in namespace for cache directory for proper destroy on restart
       it(`${testName}: deploy`, async (): Promise<void> => {
         testLogger.info(`${testName}: beginning ${testName}: deploy`);
-        await main(soloOneShotDeploy(testName));
+        await main(soloOneShotDeploy(testName, namespace));
```

Comment:
```suggestion
        await main(soloOneShotDeploy(testName, deployment));
```

Passing in the deployment is much more useful, as all of the other commands require the deployment and not the namespace.